code/game/jobs/job/civilian.dm

```diff
@@ -215,6 +215,26 @@
 		H.equip_to_slot_or_del(new /obj/item/device/pda/janitor(H), slot_belt)
 		return TRUE
 
+/datum/job/journalist
+	title = "Journalist"
+	flag = JOURNALIST
+	department = "Civilian"
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the head of personnel"
+	selection_color = "#dddddd"
+	access = list(access_journalist, access_maint_tunnels)
+	minimal_access = list(access_journalist, access_maint_tunnels)
+
+/datum/job/journalist/equip(var/mob/living/carbon/human/H)
+	if(!H)
+		return FALSE
+	H.equip_to_slot_or_del(new /obj/item/clothing/under/suit_jacket/red(H), slot_w_uniform)
+	H.equip_to_slot_or_del(new /obj/item/device/pda/librarian(H), slot_belt)
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/black(H), slot_shoes)

```

Comment:
I believe they should get a service headset (similar to the librarians)

Reply:
I'm kinda uncertain. If they're meant to be independent to some degree, you wouldn't want them on any secure channels like that anyways.

Reply:
I think my solution to this is going to be adding an alt-title. Journalists can be freelance/contracted, or corporate journalists/reporters. This would change their responsibilities and equipment accordingly- a corporate journalist would get a service headset (and presumably a press vest whenever @ParadoxSpace gets around to it), while a freelance journalist would be, effectively, an assistant with an office and the expectation to write stories.